packages/react-components/react-nav-preview/library/src/components/SplitNavItem/SplitNavItem.types.ts

```diff
@@ -1,17 +1,48 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { NavItemProps } from '../NavItem/NavItem.types';
+import { ButtonProps, MenuButtonProps, ToggleButtonProps } from '@fluentui/react-button';
+import { NavSize } from '../Nav/Nav.types';
 
 export type SplitNavItemSlots = {
+  /**
+   * Root of the component, wrapping the children.
+   */
   root: Slot<'div'>;
+
+  /**
+   * Primary navigation item in SplitNavItem.
+   */
+  primaryNavItem?: NonNullable<Slot<NavItemProps>>;
+
+  /**
+   * Secondary actions in SplitNavItem.
+   */
+  secondaryActionButton?: Slot<ButtonProps>;
```

Comment:
Q: Is 'primary/secondary' necessary here for understanding?

Reply:
I think removing them might be helpful and just having 'navButton'/'actionButton'/'toggleButton'/'menuButton' would make sense as it's more descriptive.

Are these buttons possible to use in conjunction/a combination? Do we need to provide a way to re-order or is the order standardized to these types?

Reply:
Agreed - there's a lot of  design guidance coming along with these buttons, but in a nut shell:

'Use at most 2 at a time, menu should always be on the far right." I don't know how much code I want to write to enforce this.

Reply:
or said differently - I don't really want to write any code to enforce that.

Reply:
Renamed everything a little more clearly. ✅